.travis.yml

```diff
@@ -1,17 +1,26 @@
 sudo: false
+cache: bundler
 language: ruby
 
 rvm:
-  - 2.0
-  - 2.1
-  - 2.2
-  - jruby
-  - rbx-2
+  - 2.0.0-p648
+  - 2.1.8
+  - 2.2.4
+  - 2.3.0
```

Comment:
Hmmmm, actually this seems to be an rvm problem. I just checked out one random `Reek` [PR](https://travis-ci.org/troessner/reek/jobs/108217483):

>> 0.31s$ rvm use 2.1 --install --binary --fuzzy
Using /home/travis/.rvm/gems/ruby-2.1.5
$ export BUNDLE_GEMFILE=$PWD/Gemfile
$ ruby --version
ruby 2.1.5p273 (2014-11-13 revision 48405) [x86_64-linux]

Probably nothing travis can do at this point.
I'm not the biggest fan of merging things like that but I can see the value in this PR at the same time.

@lucasmazza @Onumis objections from your side? If not, I'd go ahead and merge it.